### ISSUE TITLE
waiting for user confrrmation error shows up when trying to install from aurora store and other stores that support shizuku

### ISSUE THREAD

**Issue by nomd346** [RikkaApps/Shizuku#620]
Please report bugs of Shizuku itself.

**Requirements:**

- [ x] Shizuku version is up-to-date
- [x ] Shizuku is downloaded from official channels (GitHub release or Google Play)
- [x] Shizuku is not running in a virtual environment or broken ROM (GrapheneOS)
- [x] (Root users) No Xposed installed / Xposed is not enabled for Shizuku

- Shizuku version:latest
- Mode: adb
- Android version:12
- Device: huawei mate 40 pro 

basically i put shizuku as my install option aurora store and it was working for a while until it just stopped and said "waiting for user confirmation" and i tried other app stores and same result 

 also i didnt update or do anything before shizuku suddenly stopped
